packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/components/health-monitor-form/constants.js

```diff
@@ -0,0 +1,95 @@
+export const LABELS = {
+  MAX_RETRIES_DOWN: 'Max Retries Down',
+  MAX_RETRIES: 'Max Retries',
+  PERIODICITY: 'Periodicity',
+  TIMEOUT: 'Timeout',
+  URLPATH: 'URL Path',
+};
+
+export const MINMAX = {
```

Comment:
```suggestion
export const BOUNDS = {
```
or whatever you want but `MINMAX` is hard to read

Reply:
it comes from css minmax properties :) I replace it